src/test/java/com/salesforce/storm/spout/dynamic/VirtualSpoutTest.java

```diff
@@ -1656,9 +1764,7 @@ public void testSetEndingState() {
         defaultConfig.put(KafkaConsumerConfig.DESERIALIZER_CLASS, Utf8StringDeserializer.class.getName());
 
         // DynamicSpout config items
-        defaultConfig.put(SpoutConfig.PERSISTENCE_ZK_ROOT, "/sideline-spout-test");
```

Comment:
Removing the ZK dependency (not needed in this test) plus calling close makes this test run in less than 1 sec, compared to 1 min 30 sec+ previously. 🎈